### DISCUSSION TITLE
Welcome to Python Discussions!

### DISCUSSION THREAD

**Discussion by akshitagupta15june** [akshitagit/Python#184] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akshitagupta15june** [comment, marked as answer]
Hey everyone this is for Python Programming discussion.
Feel free to discuss anything with the community.

**Comment by Fifi0011** [comment]
как в нем вообще работать ? я могу понять что-то типо Hello world , a дальше ? есть ли какие - нибудь хитрости чтоб быстро разобраться в нем?

**Reply by akshitagupta15june** [reply]
если вы новичок в python, вы можете найти несколько бесплатных ресурсов на youtube и других платформах.
https://youtu.be/kqtD5dpn9C8
